### PR TITLE
Auto insert latest SDK version to zephyr manifest

### PR DESCRIPTION
**Pull request by noahp**
Instead of a nightly job, just auto-insert it.

### CODE REVIEW

File: docs/mcu/zephyr-guide.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ know the status!
 Adding the Memfault SDK to your project is normally as simple as adding this
 snippet to your `west.yml`:
 
-<ZephyrManifest />
+<CodeBlockWithLatestTag />
```
Comment (ejohnso49) [inline docs/mcu/zephyr-guide.mdx:50]: Shouldn't this be `ZephyrManifest` still?

Comment (noahp) [reply docs/mcu/zephyr-guide.mdx]: :upside_down_face: whoops, yes it is, fixed!

File: src/components/LatestZephyrManifest.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          remote: memfault
+  `;
+
+    // from https://docusaurus.io/docs/next/markdown-features/code-blocks#usage-in-jsx
+    return (
+        <div>
+            <CodeBlock language="yaml" title="west.yml">
+                {codeSnippet}
+            </CodeBlock>
+        </div>
```
Comment (fnune) [inline src/components/LatestZephyrManifest.js:45]: Mild case of `div`itis :grin: 
```suggestion
          <CodeBlock language="yaml" title="west.yml">
              {codeSnippet}
          </CodeBlock>
```

File: src/components/LatestZephyrManifest.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import CodeBlock from "@theme/CodeBlock";
+
+export function ZephyrManifest() {
+    const [latestTag, setLatestTag] = useState("");
```
Comment (fnune) [inline src/components/LatestZephyrManifest.js:5]:
```suggestion
    const FALLBACK_LATEST_TAG = "0.43.3"
    const [latestTag, setLatestTag] = useState(FALLBACK_LATEST_TAG);
```

Then, only `setLatestTag` in the `try` branch if there is one:

```jsx
const tag = data?.tag_name;
if (tag) {
  setLatestTag(tag);
}
```

**Review by fnune — APPROVED** [review]

**Review by fnune — APPROVED** [review]

**Pull request merged**